src/Features/CSharp/Portable/Completion/CompletionProviders/SnippetCompletionProvider.cs

```diff
@@ -157,7 +157,7 @@ SyntaxKind.EndRegionKeyword or
                 return ImmutableArray<CompletionItem>.Empty;
 
             var snippets = service.GetSnippetsIfAvailable();
-            if (context.CompletionOptions.ShouldShowNewSnippetExperience())
+            if (context.CompletionOptions.ShouldShowNewSnippetExperience() && !context.Document.IsRazorDocument())
```

Comment:
should we have an issue tracking the work needed to enable semantic snippets in razor?

Reply:
https://github.com/dotnet/razor-tooling/issues/6927

Reply:
We should probably reference that bug here then.

Reply:
Although @dibarbet and @davidwengier should we have a new bug to track fixing that? Since we'll close that bug once we merge this, right?

Reply:
I added a note to the issue about reverting this. Will need to be coordinated anyway, or we'll get doubling up of snippets.

> Since we'll close that bug once we merge this, right?

I've updated the title of that issue to better reflect the plan. We won't close the bug.